vendor/engines/split_accounts/app/services/split_accounts/journal_row_converter.rb

```diff
@@ -0,0 +1,43 @@
+module SplitAccounts
+
+  # TODO: use an instance rather than a class service object and store journal
+  # and possibly order_detail
+  class JournalRowConverter < ::JournalRowConverter
```

Comment:
Can you make this not a subclass? The trouble will be when the default converter is not `::JournalRowConverter` (like it probably will be in NU). Maybe it can look up the default converter and rely on it.